sdk/python/test_data/components/output_metrics.yaml

```diff
@@ -2,6 +2,7 @@
 # Name: output-metrics
 # Description: Dummy component that outputs metrics with a random accuracy.
 # Outputs:
+#    metrics: system.Metrics
 #    output-metrics-metrics: system.Metrics
```

Comment:
Not part of this PR, but do you know what is going on here with `output-metrics-metrics`?

Reply:
I have no idea why that is happening